.github/workflows/test-prs.yml

```diff
@@ -35,7 +35,7 @@ jobs:
             const label = 'run-tests';
             github.issues.removeLabel({ owner, repo, issue_number, name: label });
   test:
-    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@v2/react
+    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@ui-react@v2
```

Comment:
Here's the only weird part with `@` appearing twice, but if GH doesn't complain I'm good with it.